### ISSUE TITLE
Certain achievements in "Ori and the Blind Forest" revert to their unlocked state after committing changes

### ISSUE THREAD

**Issue by Braqeet** [gibbed/SteamAchievementManager#405]
So I at one point in the past I unlocked all the achievements in the game and forgot about it. But recently I wanted to replay the game again and unlock those achievements legitimately. However, as I tried to re-lock all achievements I noticed that after committing changes a certain 12 out of 50 achievements are stuck in their unlocked state. It's really weird, and I never ran into this issue before. I also tried resetting the stats in "SAM" but the same issue resides.

Anyone got any ideas as to what might be the cause of this?

**Comment by lhwe** [comment]
Yeah, it's a random issue with steam cloud saves, its annoying. I wanted to reset the achivements on a game, but since my cloud save said id unlocked a few legimiately, i couldnt.

**Comment by utbir4** [comment]
Me too.
